circup.py

```diff
@@ -687,6 +687,12 @@ def update(all):  # pragma: no cover
             click.echo("Please indicate which modules you wish to update:\n")
         for module in modules:
             update_flag = all
+            if VERBOSE:
+                click.echo(
+                    "device version:{} bundle version:{}".format(
```

Comment:
How about a little white space since this is for people to read?  Like this: `"device version: {} bundle version: {}"`